src/components/PageHeader.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './PageHeader.css'
+import PropType from 'prop-types'
+import { isValidUrl } from '../utils/index'
+
+const PageHeader = props => {
+  const generateBackground = background => ({
+    background: isValidUrl(background) ? `url(${background})` : background
+  })
+
+  return (
+    <div className="jumbotron jumbotron-fluid jumbotron" style={generateBackground(props.background)}>
```

Comment:
- you have `jumbotron` twice here
- when it comes to overriding styles, I try to follow BEM:

- `PageHeader`
- `PageHeader-title`
- `PageHeader-container`

etc..

So for the jumbotron `background-size` in your CSS, I would rename that to `PageHeader`